### DISCUSSION TITLE
is there already made waha whatsapp bot I can use?

### DISCUSSION THREAD

**Discussion by Anas099X** [devlikeapro/waha#394] [Q&A]
HI, I guess title says it all. have someone already made a bot that anyone can publicly use or do I have to create it from scratch using the waha api

**Comment by devlikepro** [comment]
Correct, from scratch, but you can use something like https://typebot.io/ to build bot (the flow) and integrate it with WAHA API

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
